app/models/location.rb

```diff
@@ -7,15 +9,23 @@ class Location < ApplicationRecord
   validates_associated :ips
 
   validates :address, presence: true
+  validates :postcode, presence: true
```

Comment:
Think you can squash the above two into a single line

validates :postcode, :address, presence: true